pkg/backend/openshiftcluster/openshiftcluster.go

```diff
@@ -40,10 +40,12 @@ type Manager struct {
 	subnet          subnet.Manager
 	acrtoken        pkgacrtoken.Manager
 
-	doc *api.OpenShiftClusterDocument
+	doc             *api.OpenShiftClusterDocument
+	subscriptionDoc *api.SubscriptionDocument
```

Comment:
I got a little confused reading this because we now have two kinds of docs, one unspecified. I'd suggest we just refer to the subscription document as `subscription` to be consistent with everything else in this struct, or explicitly give the OpenShiftClusterDocument a more descriptive name (will probably require a bunch of renaming).

Reply:
Good point. Sort of like the `cli` names, of which we have many. I think we should rename `doc` to `clusterDoc` if it becomes at all ambiguous, as it is here. (I would be against renaming `subscriptionDoc` to `subscription`, because a `SubscriptionDocument` _contains_ a field called `Subscription` which is a `*Subscription`.)

Reply:
I mean, that's clear from the type though, isn't it? Why is it meaningful to add "doc" to the name of this one and not the other fields in this struct?

Reply:
True - approaching Hungarian notation. I'll make it `subscription`, then.